### DISCUSSION TITLE
Bot is not receiving SlashCommand-Events anymore after sending 0Byte File using attachment-option

### DISCUSSION THREAD

**Discussion by MrMame** [discord-jda/JDA#2526] [Questions and Help] [answered]
Hi, 
I have some behavior of my bot, that I cannot understand. Perhaps anyone here can lead me back to track.
So thank you for reading.

**What I do :**
I've created a SlashCommand. It takes an file attachment-option which is containing text for parsing.
Everything is working as expected if the command is used as it should.
Searching for Bugs I tried to select an empty file (0 Bytes) and sending the Command.
Now I see the message "1 Files Uploading Failed" inside the channel.
The Bot is not recognizing anything - There is no Event triggered (onEvent(GenericEvent genericEvent)). 
It seems to me that the message "1 Files Uploading Failed" was send by Discord itself without sending the SlashCommand to the bot.
![image](https://github.com/discord-jda/JDA/assets/51000524/94ec12a1-8c09-461e-86a7-153c0d300c36)

**What is the Problem:**
After getting the "1 Files Uploading Failed" message I try to send another slash command to the bot. 
The "Sending command" message appears inside the channel, but the bot is not reacting on any SlashCommand Event anymore.
Other events seem to work normally. E.g. I receive the GatewayPing, ChannelCreate and other stuff.
Only SlashCommands are not working anymore.
After waiting some time, (perhaps 30 minutes i guess) the bot is working again.
![image](https://github.com/discord-jda/JDA/assets/51000524/1e37a423-057c-4631-8df8-a914bcfce782)

**Assumption:**
Could this be some kind of a bug at discord ?
It looks like they are not sending the slash commands to the bot anymore.

**I'm using:**
JDA
net.dv8tion:JDA:5.0.0-beta.12

DISCORD App on Ubuntu
Stable 222963 (3d132dc) Host 0.0.28 Linux 64-Bit (6.2.0-26-Generic)

Any idea what I'm missing here ?

regards
Markus

**Comment by MinnDevelopment** [comment, marked as answer]
Discord does not allow sending empty files. The other command not working seems to be a bug in the client, nothing you can do about that.

**Reply by MrMame** [reply]
Thank you for your answer :+1:
